### DISCUSSION TITLE
Install node shared hosting ssh

### DISCUSSION THREAD

**Discussion by madfortech** [nodejs/help#4818] [Q&A] [answered]
Any one tell me how to install node in shared hosting via ssh terminal when i install npm install
npm: command not found
Eg: [sharedhostingroot@xxx blog.example.com]$ npm install
command not found basically  i will install laravel is complete install  only node npm 
Any one help me ?

**Comment by marco-ippolito** [comment, marked as answer]
you can install [nvm](https://github.com/nvm-sh/nvm) (node version manager) through curl or wget
`curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash`
then `nvm install node`, `nvm use node`
